packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1109,7 +1111,7 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
     if (TickerMode.of(context))
       _listenToStream();
     else
-      _stopListeningToStream();
+      _stopListeningToStream(true);
```

Comment:
Make arg a named param so we know here what's set to true?

Reply:
Done